File: python/test/test_extrema.py
import ThrustRTC as trtc



darr = trtc.device_vector_from_list([ 1, 0, 2, 2, 1, 3], 'int32_t')
print (trtc.Min_Element(darr))
print (trtc.Max_Element(darr))
print (trtc.MinMax_Element(darr))

File: java/test/test_extrema.java
import java.util.*;
import JThrustRTC.*;

public class test_extrema
{
	public static void main(String[] args) 
	{
        DVVector d_data = new DVVector(new int[] { 1, 0, 2, 2, 1, 3 });
        System.out.println(TRTC.Min_Element(d_data));
        System.out.println(TRTC.Max_Element(d_data));
        System.out.println(Arrays.toString(TRTC.MinMax_Element(d_data)));
	}
}
